GameDefinition/Content/ItemKindOrgan.hs

```diff
@@ -41,15 +41,15 @@ fist = ItemKind
   , iaspects = []
   , ieffects = []
   , ifeature = [Durable, Identified, Meleeable]
-  , idesc    = ""
+  , idesc    = "Simple but effective."
```

Comment:
The organs are terrific. You really nail it with the brevity. We will show the organ descriptions more, perhaps as a part of organ descriptions. Right now, they are mentioned when they are used in whatever way and then can be checked out in the @ menu for player's own organs and in the ~ menu for all, including enemy's.

Reply:
Filed in https://github.com/LambdaHack/LambdaHack/issues/99#issuecomment-352255307